django/applications/catmaid/static/js/widgets/treenode-table.js

```diff
@@ -382,7 +382,8 @@
       if (event.which == 13) {
         // Filter with a regular expression
         this.filter_searchtag = $('#search_labels' + this.widgetID).val();
-        this.oTable.fnFilter(this.filter_searchtag, 2, true);
```

Comment:
This is completely fine for this pull request, but in general such changes should be in a separate commit (since they are unrelated to the other changes).